### PR TITLE
Storagenode online confirmation ping

### PR DESCRIPTION
**Pull request by dylanlott**
Lays out a simple online / offline check against the bootstrap node that appears on the dashboard

### CODE REVIEW

File: cmd/storagenode/main.go [hunk 1]
```diff
@@ -387,7 +399,7 @@ func dashCmd(cmd *cobra.Command, args []string) (err error) {
 
 		fmt.Fprintf(color.Output, "Node ID: %s\n", color.YellowString(data.GetNodeId()))
 
-		if data.GetConnection() {
+		if online {
```
Comment (wthorp) [inline cmd/storagenode/main.go]: Could we call this on a timer instead, in case of a temporary outage?

Comment (dylanlott) [reply cmd/storagenode/main.go]: I'm gonna put it on a ticker 👍

File: cmd/inspector/main.go [hunk 2]
```diff
@@ -131,7 +131,7 @@ type Inspector struct {
 // and the overlay cache
 func NewInspector(address, path string) (*Inspector, error) {
 	ctx := context.Background()
-	identity, err := identity.Config{
+	id, err := identity.Config{
```
Comment (egonelbre) [inline cmd/inspector/main.go]: nit: `id` in the codebase usually refers to `NodeID`, not the identity.

**Comment by dylanlott** [comment]
I can update it to “fid” or something. Identity just felt wrong because it’s also a package name.   
  

  
  

  
  
>

**Comment by egonelbre** [comment]
@dylanlott there are places `ident` is used, although I personally prefer and often use `identity`.

**Pull request merged**